plugins.json

```diff
@@ -1298,5 +1298,10 @@
         "description": "Properly align link underline layers regardless of their text in Sketch",
         "name": "sketch-link-underline-align",
         "owner": "zholdas"
+    },
+    {
+        "description" : "Show QR Code for Local Sharing",
+        "name" : "QRCode for Local Sharing",
```

Comment:
This thing here is missing some dashes, right? It should be `QRCode-for-Sketch-Local-Sharing`